.cirrus.yml

```diff
@@ -1,13 +1,30 @@
 task:
   name: Test on Ventura
+  alias: test
   persistent_worker:
     labels:
-      name: Mac-Mini-M1
-  build_script: swift test
-  test_script: swift test
+      name: scaleway-m1
+  test_script:
+    - swift test
+  integration_test_script:
+    # Build Tart
+    - swift build
+    - codesign --sign - --entitlements Resources/tart.entitlements --force .build/debug/tart
+    - export PATH=$(pwd)/.build/arm64-apple-macosx/debug:$PATH
+    # Run integration tests
+    - cd integration-tests
+    - brew install virtualenv
```

Comment:
I'm not a fan of this approach because it [harms reproducibility](https://github.com/cirruslabs/tart/pull/207#discussion_r957618273).

If you're worried about build times, we can use `HOMEBREW_NO_AUTO_UPDATE=1` to speed it up a little, but still: this particular step takes very little time compared to the overall run time of the integration tests.

Reply:
Added `HOMEBREW_NO_AUTO_UPDATE=1` in https://github.com/cirruslabs/tart/pull/313/commits/1b7ae4a1d0048c299f2faa71cafd2dfcfb57cd14.